### DISCUSSION TITLE
Discrepancy in Completed Kata Counts

### DISCUSSION THREAD

**Discussion by ricardoagmg** [codewars/codewars.com#2971] [Q&A]
I noticed that there is a discrepancy in the number of completed kata shown on my profile. Specifically, the "Total Completed Kata" count on my profile shows 149, but the "Completed Kata" count in the "Honor Breakdown" section shows 344. I also recently completed several new kata, but these were only added to the "Completed Kata" count in the "Honor Breakdown" section, and not to the "Total Completed Kata" count on my profile.

I'm not sure why there's a difference between the two counts, and why the new kata I completed were not included in the "Total Completed Kata" count. I'd appreciate it if you could help me figure out what's going on.

I have double-checked my profile and made sure that all of the kata I've completed have been submitted and passed all of the tests. I'm also training in the same language (JavaScript) as the majority of the kata I'm completing.

My profile  : https://www.codewars.com/users/ricardoag1

**Comment by hobovsky** [comment]
"Total Completed Kata" is amount of kata you solved. It's also cached, and gets refreshed once a day or so. Changes to this stat might be not visible immediately.

"Honor Breakdown" shows amount of honor points you earned by solving the kata.

So what you see basically means that you solved 149 kata and earned 344 Honor for solving them.

**Reply by ricardoagmg** [reply]
Now I understand how it works, thank you for responding.
